test-scripts/entrypoint

```diff
@@ -186,6 +192,8 @@ if [ "x$2" == "xserver" ]; then
     _cpus=$9
     _vnodes=${10}
     if [ "x${_moms}" != "xnone" ]; then
+    # delete nodes created automatically.
+	PBS_SERVER_INSTANCES=${_svr}:$5 /opt/pbs/bin/qmgr -c "d n @default"
```

Comment:
this doesn't work right? you were getting the object busy error? let's not check it in then